src/IIIFPresentation/API/Converters/CollectionConverter.cs

```diff
@@ -132,36 +132,33 @@ public static List<string> GenerateContext()
     /// </summary>
     /// <param name="collection">The database collection to use</param>
     /// <returns>A list of behaviors</returns>
-    public static List<string>? GenerateBehavior(Models.Database.Collections.Collection collection)
+    public static List<string>? GenerateBehavior(DbCollection collection)
     {
         var behaviours = new List<string>()
             .AppendIf(collection.IsPublic, Behavior.IsPublic)
             .AppendIf(collection.IsStorageCollection, Behavior.IsStorageCollection);
-        
-        return behaviours.Any() ? behaviours : null; 
+
+        return behaviours.Any() ? behaviours : null;
     }
 
     /// <summary>
     /// Generates the PartOf field for a collection
     /// </summary>
-    /// <param name="hierarchy">The hierarchy to use to generate</param>
     /// <param name="collection">The collection required</param>
     /// <param name="pathGenerator">Class used to generate paths for collections</param>
     /// <returns>A list of ResourceBase</returns>
-    public static List<ResourceBase>? GeneratePartOf(Hierarchy hierarchy, Models.Database.Collections.Collection collection, 
-        IPathGenerator pathGenerator)
-    {
-        return hierarchy.Parent != null
-            ? new List<ResourceBase>
-            {
-                new PartOf(nameof(PresentationType.Collection))
+    public static List<ResourceBase>? GeneratePartOf(DbCollection? collection,
+        IPathGenerator pathGenerator) =>
+        collection != null
+            ?
+            [
+                new ExternalResource(nameof(PresentationType.Collection))
                 {
-                    Id = pathGenerator.GenerateHierarchicalCollectionParent(collection, hierarchy),
-                    Label = collection.Label
+                    Id = pathGenerator.GenerateFlatCollectionId(collection),
```

Comment:
Should this be changed to `GeneratePresentationCollectionId` given the rename

Reply:
I don't think so, renaming `ToFlatCollection()` -> `ToPresentationCollection()` is to match the return type (`FlatCollection` was renamed to `PresentationCollection` in #47).

I think the flat methods on `IPathGenerator` still make sense as the paths are flat (or hierarchical). If we want to revisit we can do and rename all methods in `IPathGenerator` that refer to 'flat' paths.